common/locales/en/contrib.json

```diff
@@ -54,8 +54,9 @@
     "playerTiers": "Player Tiers",
     "tier": "Tier",
     "visitHeroes": "Visit the Hall of Heroes (contributors and backers)",
-    "conLearn": "Learn more about contributor rewards",
+    "conLearn": "<a href='http://habitica.wikia.com/wiki/Contributor_Rewards' target='_blank'>Learn more about contributor rewards</a>",
```

Comment:
I'm :+1: on a naming convention for things that are just links versus things that contain links.